.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.11.0
+    resource_class: large
+    working_directory: ~/sane-reports
+    steps:
+      - checkout
+      - restore_cache:
+          key: npm-sane-reports-{{ checksum "package.json" }}
+      - run:
+          name: Install dependencies
+          command: |
+              sudo npm install npm@4 -g
```

Comment:
@Bars92 why not latest?

Reply:
npm 5 and 6 have some issues with our configurations that cause npm install to hang https://circleci.com/gh/demisto/sane-reports/234. Managed to fix in circleci 2 for the client i will try later here too